pkgs/development/python-modules/pysol_cards/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, python3Packages }:
+
+buildPythonPackage rec {
+  pname = "pysol_cards";
```

Comment:
```suggestion
  pname = "pysol-cards";
```

Reply:
The upstream package seems to actually be called `pysol_cards` - see https://pypi.org/project/pysol-cards/#files. If I change the pname here to `pysol-cards` I'd have to override it in the `fetchPypi` block. Thoughts?

Reply:
Yes, we [use](https://github.com/NixOS/nixpkgs/commit/4af35d92a0ceee11b8082befccbba2b036f5a536) `-` in python package names even if upstream uses `_`. Both `pysol_cards` and `pysol-cards` works when using `pip` to install packages.

Reply:
Got it. How's that look?